recipes/UniFunc/meta.yaml

```
@@ -0,0 +1,47 @@
+
+{% set name = "unifunc" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PedroMTQ/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
```

```suggestion
  url: https://github.com/PedroMTQ/UniFunc/archive/refs/tags/{{ version }}.tar.gz
```
This is sufficient, the name shouldn't change so no placeholder required.